kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties

```diff
@@ -14,3 +14,5 @@ loginAsAnotherUser=Connectez-vous en tant qu'un autre utilisateur
 logoutSuccssful=D\u00e9connexion r\u00e9ussie
 # translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author croe@redhat.com
 loginAgain=Connectez-vous \u00e0 nouveau
+loadingPleaseWait=Veuillez patienter
+loadingApplication=Chargement de l'application...
```

Comment:
reviewing the script, for kie-wb.distribution it does:
mvn -B zanata:pull-module
mvn replacer:replace -N
mvn native2ascii:native2ascii followed but
mvn -B zanata:pull-module -Dproductized
mvn replacer:replace -N -Dproductized
mvn native2ascii:native2ascii -Dproductized

I can't see something wrong there.